seatgeek/SGAPI
# MOBILE-17733 Audit Files in the SDK

### Issue ⚠️

[MOBILE-17733](https://seatgeek.atlassian.net/browse/MOBILE-17733)

### Description 📓

This PR renames `SGEvent` to `SGKEvents` to avoid conflicts within our SDK. There exists a [framework](https://developer.limneos.net/index.php?ios=13.1.3&framework=CoreSuggestions.framework&header=SGEvent.h) with the same name and though we have not had any conflicts yet, it is safer to change the name.

### How to Test 🧪

Note: The test project doesn't seem to be able to build, but I believe we can test the changes with our `iphone-app` project

1. Check that all instances of `SGEvent` have been renamed
